home_assistant_streamdeck_yaml.py

```diff
@@ -1248,8 +1605,36 @@ def _max_filter(value: float, other_value: float) -> float:
     """
     return max(value, other_value)
 
+def _round(num, digits):
+    """Returns rounded value with number of digits"""
+    res = round(num, digits)
+    return res
+
+def _int(num) -> int:
```

Comment:
why do we need to wrap `int`?  
if we need it, add missing type annotation for num argument.